CHANGELOG-unreleased.md

```diff
@@ -9,6 +9,25 @@ the released changes.
 
 ## Unreleased
 ### Changed
+- Avoided unnecessary creation of `SkyCoord` objects in `AstrometryEquatorial` and `AstrometryEcliptic`.
+- Avoided unnecessary `TOAs` table slices in `SolarSystemShapiro`
+- Allow "CLK UNCORR" in par files (indicates no GPS or BIPM corrections). 
+- Better documentation for `akaike_information_criterion()`
+- Type hinting for most of the `pint.utils` module
+- `funcParameter`s are no longer listed in the `pintk` interface.
+- Updated location of CCERA
+- Removed `include_bipm`, and `bipm_version` from the Observatory class. Now they are passed as arguments to `site.clock_correction()`
+- Renamed `include_gps` to `apply_gps2utc` in the Observatory class
+- Removed `apply_gps2utc` from `TOAs.clock_corr_info` since it can be different for different observatories. It is not a global setting.
+- The following observatories no longer have a default of `include_bipm=False`: magic, lst, virgo, lho, llo, geo600, kagra, hess, hawc
+- New algorithm for TCB <-> TDB conversion
+- Reordered plotting axes in `pintk`
+- Changed `scipy.integrate.simps` to `scipy.integrate.simpson` to work with scipy 1.14
```

Comment:
Please delete these lines. They have been moved from this file when we tagged the new version.

Reply:
Updated! Thanks for pointing that out